src/components/review-components/reviews/reviews.component.jsx

```diff
@@ -22,22 +22,24 @@ class Reviews extends React.Component {
 
   refresh = () => {
     const { id } = this.props;
-    const { sort, count } = this.state;
-    fetch(`http://3.134.102.30/reviews/${id}/list?count=${count}&sort=${sort}`)
+    const { sort } = this.state;
+    fetch(`http://3.134.102.30/reviews/${id}/list?count=1000&sort=${sort}`)
       .then((data) => data.json())
       .then((res) => this.setState({ reviews: res.results }))
       .catch((err) => err);
   }
 
-  handleChange = async (e) => {
-    await this.setState({ sort: e.target.value });
-    this.refresh();
+  handleChange = (e) => {
+    this.setState({ sort: e.target.value }, () => {
+      this.refresh();
+    });
   }
 
-  loadMoreReviews = async () => {
+  loadMoreReviews = () => {
     const { count } = this.state;
-    await this.setState({ count: count + 2 });
-    this.refresh();
+    this.setState({ count: count + 2 }, () => {
+      this.refresh();
```

Comment:
No longer need to use refresh if your count is 1000

Reply:
good eye sir, good eye